package.json

```
@@ -10,7 +10,7 @@
   "scripts": {
     "test": "echo \"Error: no test specified\" && exit 1"
   },
-  "author": "Bart Siniarski <bartlomiej_siniarski@rapid7.com>",
+  "author": "Szymon Bialkowski <szymon_bialkowski@rapid7.com>",
```

```suggestion
  "author": "Rapid7 <InsightOpsTeam@rapid7.com>",
```

https://github.com/rapid7/r7insight_python/pull/3/files#diff-2eeaed663bd0d25b7e608891384b7298R12

---

Was wondering about this